revup/types.py

```diff
@@ -73,6 +73,10 @@ class RevupGithubException(Exception):
     def __init__(self, error_json: Dict):
         super().__init__()
         self.error_json = error_json
+        self.message = ""
+        for error in self.error_json:
+            self.type = error["type"] if "type" in error else "Unknown Error"
+            self.message += error["message"]
```

Comment:
Do we need newlines between these?  Also isn't this going to override the type, is that going to be confusing?